### PR TITLE
Update 17-shinshin0923.md

### LINKED ISSUE #12: shinshin0923 Workflow
YOUR-USERNAMEは画像を選択し、キャプションを追加し、両方をファイルに追加します。

- [ ] ブランチの作成
- [ ] ファイルの変更
- [ ] 変更のコミット
- [ ] Pull Requestの作成
- [ ] レビューのリクエスト
- [ ] 変更を追加
- [ ] 承認を受ける
- [ ] Pull Requestをマージ

### PR DESCRIPTION
**Pull request by shinshin0923**
resolves #12

### CODE REVIEW

File: _slides/17-shinshin0923.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is Test
```
Comment (Gentaro-I) [inline _slides/17-shinshin0923.md:6]: Good!

Comment (Gentaro-I) [inline _slides/17-shinshin0923.md:6]: Awesome

Comment (Gentaro-I) [inline _slides/17-shinshin0923.md:6]: Great

**Review by Gentaro-I — APPROVED** [review]

**Comment by shinshin0923** [comment]
merge!

**Pull request merged**